### PR TITLE
Demo Changes added.

### PR DESCRIPTION
**Pull request by Nikita5796**

### CODE REVIEW

File: src/app/customer/customer.service.ts [hunk 1]
```diff
@@ -4,28 +4,16 @@ import { Observable } from 'rxjs';
...
-  
-    constructor(private http: HttpClient) {}
 
-  doRegistration(customerRegistration){
-      console.log(customerRegistration);
-      return this.http.post("https://localhost:44378/api/users/add"+customerRegistration, {responseType : 'text' as 'json'});
+  constructor(private http: HttpClient) { }
+
+  doRegistration(customerRegistration) {
+    console.log(customerRegistration);
```
Comment (bala0105) [inline src/app/customer/customer.service.ts]: remove console.log

File: src/app/customer/login/login.component.ts [hunk 2]
```diff
@@ -2,33 +2,33 @@ import { Component } from '@angular/core';
...
-
-    response.subscribe((data) =>
-    console.log(data)
-      
-    );
-    alert("successfully Logged in");
-    this.router.navigateByUrl('booklist');
-
+    response.subscribe((data: any) => {
+      console.log(data.token);
```
Comment (bala0105) [inline src/app/customer/login/login.component.ts]: same here

File: src/app/customer/customer.service.ts (hunk 1 above)
Comment (Nikita5796) [reply src/app/customer/customer.service.ts]: done.

File: src/app/customer/login/login.component.ts (hunk 2 above)
Comment (Nikita5796) [reply src/app/customer/login/login.component.ts]: done

File: src/app/customer/login/login.component.ts [hunk 3]
```diff
@@ -2,33 +2,32 @@ import { Component } from '@angular/core';
...
   templateUrl: './login.component.html',
 })
 export class LoginComponent {
 
-  customer : customer = new customer();
+  customer: customer = new customer();
+  isLoginError: boolean = false;
 
-  constructor(private service1 : CustomerService, private router : Router){}
+  constructor(private service1: CustomerService, private router: Router) { }
```
Comment (ketannaik) [inline src/app/customer/login/login.component.ts]: Why is the variable name service1??
Please follow proper naming conventions

File: src/app/home/home.component.ts [hunk 4]
```diff
@@ -9,7 +9,12 @@ import { ActivatedRoute, ParamMap } from '@angular/router';
   templateUrl: './home.component.html',
 })
 export class HomeComponent {
-  constructor(){}
+  constructor(private router: Router) { }
 
-  ngOnUnit(){}
+  ngOnUnit() { }
+
+  Logout() {
```
Comment (ketannaik) [inline src/app/home/home.component.ts]: Function name should be in camel Case

File: src/app/customer/login/login.component.ts [hunk 5]
```diff
@@ -2,33 +2,32 @@ import { Component } from '@angular/core';
...
+
+  ngOnUnit() {
```
Comment (ketannaik) [inline src/app/customer/login/login.component.ts]: What is ngOnUnit?

File: src/app/home/home.component.ts [hunk 6]
```diff
@@ -9,7 +9,12 @@ import { ActivatedRoute, ParamMap } from '@angular/router';
   templateUrl: './home.component.html',
 })
 export class HomeComponent {
-  constructor(){}
+  constructor(private router: Router) { }
 
-  ngOnUnit(){}
+  ngOnUnit() { }
```
Comment (ketannaik) [inline src/app/home/home.component.ts]: Even here

**Review by ketannaik — CHANGES_REQUESTED** [review]

File: src/app/customer/login/login.component.ts (hunk 3 above)
Comment (Nikita5796) [reply src/app/customer/login/login.component.ts]: renamed service1 with customerService.

File: src/app/home/home.component.ts (hunk 4 above)
Comment (Nikita5796) [reply src/app/home/home.component.ts]: done.

File: src/app/customer/login/login.component.ts (hunk 5 above)
Comment (Nikita5796) [reply src/app/customer/login/login.component.ts]: replaced with ngOnInit().

File: src/app/home/home.component.ts (hunk 6 above)
Comment (Nikita5796) [reply src/app/home/home.component.ts]: changes done here too.

**Review by ketannaik — APPROVED** [review]

**Pull request merged**